<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealWarningItem</class>
 <widget class="QDialog" name="RealWarningItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid rgb(180,180,180)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_backWidget" native="true">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>告警照片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>相似度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>对比照片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelWarningPlace">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>162</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>告警地点</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>告警时间</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_checkBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>6</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelWarningTime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>告警时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>162</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>告警地点</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>184</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>区域</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>74</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>年龄</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelAge">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>74</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>年龄</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelName">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>29</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelImage1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelImportanceLevel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>96</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>重要等级</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>96</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>重要等级</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelDangersLevel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>118</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>危险等级</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_btnDispos">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>处理</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>118</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>危险等级</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelImage2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>81</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelSex">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>51</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelArea">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>184</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>区域</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>52</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelSimilar">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 9pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>99.99%</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
